modules/frontend/src/lib/history/method-badge.svelte

```diff
@@ -1,39 +1,25 @@
 <script lang="ts">
+  import Badge from '$lib/values/util/badge.svelte';
+
   interface Props {
     method: 'GET' | 'HEAD' | 'POST' | 'PUT' | 'DELETE' | 'PATCH';
   }
 
   let { method }: Props = $props();
 
-  function modeBgColor(mode: string): string {
-    switch (mode) {
-      case 'POST':
-        return 'bg-green-100';
-      case 'PUT':
-        return 'bg-fuchsia-100';
-      default:
-        return 'bg-red-100';
-    }
-  }
-
-  function modeTextColor(mode: string): string {
-    switch (mode) {
+  function methodColor(method: string): string {
+    switch (method) {
       case 'POST':
-        return 'text-green-800';
+        return 'green';
       case 'PUT':
-        return 'text-fuchsia-800';
+        return 'fuchsia';
       default:
```

Comment:
the default should come last and return something or you need to change the return type to `string | undefined`